### PR TITLE
Update to SDK 1.4.0.

### PR DESCRIPTION
**Pull request by slaff**
For details see: http://bbs.espressif.com/viewtopic.php?f=46&t=1124

### CODE REVIEW

**Comment by slaff** [comment]
@pfalcon Hi Paul, there is a new SDK version with Mesh stuff merged in.

**Comment by pfalcon** [comment]
Thanks. I'm travelling currently, so cannot test it well. I might either merge it, but keep default at 1.3.0, or wait until later.

**Comment by pastukhov** [comment]
```
artem@artem-desktop-bt:~/build/esp-open-sdk$ make STANDALONE=n
Makefile:78: *** missing separator.  Останов.
````

File: Makefile [hunk 1]
```diff
@@ -71,6 +73,10 @@ libcirom: $(TOOLCHAIN)/xtensa-lx106-elf/sysroot/lib/libcirom.a
 
 sdk_patch: .sdk_patch_$(VENDOR_SDK)
 
+.sdk_patch_1.4.0:
+	patch -N -d $(VENDOR_SDK_DIR_1.4.0) -p1 < c_types-c99.patch
+        @touch $@
```
Comment (eku) [inline Makefile]: Please correct whitespaces type. Tabulator required.

**Comment by slaff** [comment]
Hi @pfalcon , can you check  if the spacing is ok now?

**Comment by pastukhov** [comment]
Yes. Now it's ok.

**Comment by pfalcon** [comment]
Merged, thanks.

**Pull request closed without merging**